dll/win32/shell32/shresdef.h

```diff
@@ -153,6 +153,9 @@
 #define IDS_CANTDISCONNECT        160
 #define IDS_NONE                  161
 
+#define IDS_EXPAND 170
+#define IDS_COLLAPSE 171
```

Comment:
Following the ambient formatting of the file:
```suggestion
#define IDS_EXPAND                170
#define IDS_COLLAPSE              171
```